app/models/user_language.rb

```diff
@@ -1,4 +1,9 @@
 class UserLanguage < ApplicationRecord
+  include ULID::Rails
+  ulid :id, auto_generate: true
+  ulid :user_id
+  ulid :language_id
+
```

Comment:
Spent way too long trying to debug why `user.languages` or `language.users` wasn't working. This is needed for has many through relationships

https://github.com/k2nr/ulid-rails#many-to-many-associations